mona_sdk/client.py

```diff
@@ -656,7 +662,8 @@ def get_suggested_config(self):
         """
         app_server_response = self._app_server_request("get_new_config_fields")
         try:
-            return app_server_response["response_data"]["suggested_config"]
+            data = app_server_response["response_data"]["suggested_config"]
+            return get_dict_result(True, f"suggested_config: {data}", None)
```

Comment:
why this prefix? this is an api call, I'd expect it to be returned as an object {data} as is, am I missing something?

Reply:
This prefix unnecessary I removed it